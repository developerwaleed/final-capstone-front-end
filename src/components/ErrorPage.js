import React from 'react';

const ErrorPageStyles = {};

const ErrorPage = () => (
  <div className="error__page container p-2 m-0 w-100 h-100 bg-light d-flex flex-column justify-content-center" style={ErrorPageStyles}>
    <h1 className="text-center fw-bold">404</h1>
    <p className="text-center">I&apos;m sorry but we couldn&apos;t find the page you&apos;re looking for</p>
  </div>
);

export default ErrorPage;
