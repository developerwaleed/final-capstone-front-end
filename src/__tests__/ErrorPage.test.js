import renderer from 'react-test-renderer';
import ErrorPage from '../components/ErrorPage';

it('<ErrorPage /> renders correctly', () => {
  const tree = renderer
    .create(<ErrorPage />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
