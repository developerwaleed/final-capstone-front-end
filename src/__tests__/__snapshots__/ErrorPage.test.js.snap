// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorPage /> renders correctly 1`] = `
<div
  className="error__page container p-2 m-0 w-100 h-100 bg-light d-flex flex-column justify-content-center"
  style={Object {}}
>
  <h1
    className="text-center fw-bold"
  >
    404
  </h1>
  <p
    className="text-center"
  >
    I'm sorry but we couldn't find the page you're looking for
  </p>
</div>
`;
